data/Create_data/api_football/get_fixtures_postgre.py

```
@@ -527,75 +638,94 @@ def get_statistics(self, fixture_id: int) -> dict:
                         try:
-                            val = float(val.replace('%', ''))
+                            val = float(val.replace("%", ""))
                         except Exception:
                             val = None
                     # Convert passes_percent from '55%' to float if needed
-                    if k == 'passes_percent' or k == 'passes_%' and isinstance(val, str) and val.endswith('%'):
+                    if (
+                        k == "passes_percent"
+                        or k == "passes_%"
+                        and isinstance(val, str)
+                        and val.endswith("%")
```

Ruff (0.13.3)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)